constants/reforgestones.json

```diff
@@ -290,6 +290,27 @@
       }
     }
   },
+  "BOO_STONE": {
+    "internalName": "BOO_STONE",
+    "reforgeName": "Greater Spook",
+    "reforgeType": "blacksmith/reforge_stone",
+    "itemTypes": "ARMOR",
+    "requiredRarities": [
+      "EPIC",
+      "LEGENDARY"
+    ],
+    "reforgeCosts": {
+      "EPIC": 10000,
+      "LEGENDARY": 10000
+    },
+    "reforgeAbility": "Grants §a+1 §5Fear§7.",
+    "reforgeStats": {
```

Comment:
Please remove reforgeStats as its empty

Reply:
Was unsure if it worked without reforgeStats. But tested it now and nothing bad happend so it should be fine without it.